### PR TITLE
libimobiledevice: fix compiling with cython support

### PR DESCRIPTION
**Pull request by dhilipsiva**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

Edit the `libimobiledevice` Formulae to add support for compiling with cython.

### CODE REVIEW

**Comment by tdsmith** [comment]
What does building with or without Cython end up doing?

**Comment by dhilipsiva** [comment]
If you compile it with Cython support - you will be able to import it programatically  in python. It basically compiles with python support. This is required if you want programatically access libimobiledevice from python.

File: Library/Formula/libimobiledevice.rb [hunk 1]
```diff
@@ -26,14 +26,20 @@ class Libimobiledevice < Formula
   depends_on "usbmuxd"
   depends_on "openssl"
 
+  option "with-cython", "Compile with Cython support - for Python bindings"
+
   def install
+    opts = []
+    if build.without? "with-cython"
+      # As long as libplist builds without Cython
+      # bindings, libimobiledevice must as well.
```
Comment (mikemcquaid) [inline Library/Formula/libimobiledevice.rb]: In that case you need a conditional `depends_on "libplist" => "with-python"`.

File: Library/Formula/libimobiledevice.rb [hunk 2]
```diff
@@ -26,14 +26,20 @@ class Libimobiledevice < Formula
   depends_on "usbmuxd"
   depends_on "openssl"
 
+  option "with-cython", "Compile with Cython support - for Python bindings"
```
Comment (mikemcquaid) [inline Library/Formula/libimobiledevice.rb]: Looks from `libplist.rb` that this should be `with-python` and have a `depends_on :python => :optional`

File: Library/Formula/libimobiledevice.rb [hunk 3]
```diff
@@ -26,14 +26,20 @@ class Libimobiledevice < Formula
...
+      opts << "--without-cython"
+    end
     system "./autogen.sh" if build.head?
     system "./configure", "--disable-dependency-tracking",
                           "--disable-silent-rules",
                           "--prefix=#{prefix}",
-                          # As long as libplist builds without Cython
-                          # bindings, libimobiledevice must as well.
-                          "--without-cython"
+                          *opts
```
Comment (mikemcquaid) [inline Library/Formula/libimobiledevice.rb]: Name this `args`

**Comment by dhilipsiva** [comment]
Holy cow! Made some really silly mistake why trying to push changes @mikemcquaid suggested. Will Close this PR now. Will send an another proper PR. :stuck_out_tongue:

**Pull request closed without merging**